js/notify.js

```diff
@@ -1,30 +1,23 @@
 /**
  * User notification module
  */
-define([
-    'jquery',
-    'logger',
-    'config',
-    'templates/notification.hbs'
-], function($, Log, config, message_template) {
-    'use strict';
+import $ from 'jquery';
+import log from 'logger';
+import config from 'config';
+import message_template from 'templates/notification.hbs';
 
-    function show_message(message, type, container) {
-        container = container || config.notify_in;
-        $(container).prepend(message_template({
-            level: type == 'error' ? 'danger' : type,
-            message: message
-        }));
-        Log.debug('Notify:', {type: type, message: message, container: container});
-    }
+function show_message(message, type, container) {
+    container = container || config.notify_in;
+    $(container).prepend(message_template({
```

Comment:
It must depends on the browser because I tried with an empty element eand it works (the second `insertBefore()` parameter is optional).
If you have a working solution with `insertAdjacenHTML`, just go with it. It was only a suggestion ;)